workflow/scripts/create_modelrun.py

```diff
@@ -318,4 +322,4 @@ def main(
         with open(sys.argv[4], "r") as f:
             user_config = _read_file(f, ending)
 
-        main(sys.argv[1], sys.argv[2], sample, user_config)
+        main(sys.argv[1], sys.argv[2], sample, user_config, sys.argv[5])
```

Comment:
Suggest you wrap sys.argv[5] with `int(sys.argv[5])`

Reply:
addressed